app/src/main/java/com/example/sprint8/UI/fragments/SearchFragment.kt

```diff
@@ -73,10 +76,10 @@ class SearchActivity : AppCompatActivity() {
             inputEditText?.setText("")
 
             val inputMethodManager =
-                getSystemService(Context.INPUT_METHOD_SERVICE) as? InputMethodManager
+                activity?.getSystemService(Context.INPUT_METHOD_SERVICE) as? InputMethodManager
```

Comment:
🍏 getSystemService можно вызывать на том контексте, который доступен во фрагменте
```suggestion
                requireContext().getSystemService(Context.INPUT_METHOD_SERVICE) as? InputMethodManager
```